apax/md/io.py

```diff
@@ -51,7 +51,9 @@ def atoms_from_state(self, state, energy, nbr_kwargs):
         atoms = Atoms(self.atomic_numbers, positions, momenta=momenta, cell=box)
         atoms.cell = atoms.cell.T
         atoms.pbc = np.diag(atoms.cell.array) > 1e-6
-        atoms.calc = SinglePointCalculator(atoms, energy=float(energy), forces=forces)
+        predictions = {k: np.array(v) for k, v in predictions.items()}
+        predictions["energy"] = predictions["energy"].item()
```

Comment:
Is this line necessary? And what is it doing?

Reply:
It's the exact same line of code as in the ASE calculator. We need to loop through the dictionary and convert the values to np from jnp. The energy comes out as an array of shape (1,) instead of a float of shape (). This causes problems in ASE for the integrators and here when saving to H5.
`.item()` does this conversion.

Reply:
> Is this line necessary? And what is it doing?

It's the exact same line of code as in the ASE calculator. We need to loop through the dictionary and convert the values to np from jnp. The energy comes out as an array of shape (1,) instead of a float of shape (). This causes problems in ASE for the integrators and here when saving to H5.
`.item()` does this conversion.

Reply:
ok perfect. Thank you.